core/domain/storage_model_audit_jobs_test.py

```diff
@@ -48,6 +48,9 @@
     'ExplorationAnnotationsModel',
     'StateAnswersModel',
     'StateAnswersCalcOutputModel',
+    'BlogPostModel',
+    'BlogPostRightsModel',
+    'BlogPostSummaryModel'
```

Comment:
Proper validations for all three models will be written in the next PR in which domain and services will also be written.

Reply:
I have added validations such as `_get_external_id_relationships` and `_get_model_id_regex`.  ANd removed these models from this list here. However other validations like `_get_model_domain_object_instance` or validations which will see if the blog post are published or not ( like thumbnail file is present and summary is not empty) can only be added once the blog_service and blog_domain file is written. So, I will add these validations in the next PR.

I hope this will work.